docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,428 @@
+2020-08-25 13:32:13 Using default protocol ID "sup" because none is configured in the chain specs
+2020-08-25 13:32:13 🏷  Local node identity is: 12D3KooWC9nCQBJeL9gdeQr87mJMJeJKRydVpTSxyD7oPb62N42U (legacy representation: QmPsncwgCNBtopg2V1JR9DpicZ4jQAQR2M4qWQ72JLmKRT)
+2020-08-25 13:32:13 📦 Highest known block at #0
+2020-08-25 13:32:15 🔨 Initializing Genesis block/state (state: 0x336a…4bf1, header-hash: 0xe482…3e12)
+2020-08-25 13:32:15 👴 Loading GRANDPA authority set from genesis on what appears to be first startup.
+2020-08-25 13:32:15 ⏱  Loaded block-time = 6000 milliseconds from genesis on first-launch
+2020-08-25 13:32:15 👶 Creating empty BABE epoch changes on what appears to be first startup.
+2020-08-25 13:32:15 🏷  Local node identity is: 12D3KooWG5vyoKZggwpEqee8PqNqp6RYFh9gJgPWYAXMFx2ZPcGU (legacy representation: QmeVUPFTAqDBRvEMyi8XATMbeF9ykH5txPgWooRzSyoDZE)
+2020-08-25 13:32:15 📦 Highest known block at #0
+2020-08-25 13:32:15 〽️ Prometheus server started at 127.0.0.1:9616
+2020-08-25 13:32:15 📪 Libp2p listener () closed: Address already in use (os error 48)
+2020-08-25 13:32:15 📪 Libp2p listener () closed: Address already in use (os error 48)
+2020-08-25 13:32:18 ✨ [Relaychain] Imported #268 (0xb6ba…65e0)
+2020-08-25 13:32:18 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:20 💤 [Relaychain] Idle (3 peers), best: #268 (0xb6ba…65e0), finalized #265 (0x081c…1b35), ⬇ 18.1kiB/s ⬆ 2.4kiB/s
+2020-08-25 13:32:23 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:24 ✨ [Relaychain] Imported #269 (0x2dd2…9964)
+2020-08-25 13:32:25 💤 [Relaychain] Idle (3 peers), best: #269 (0x2dd2…9964), finalized #266 (0xf093…fd8f), ⬇ 1.0kiB/s ⬆ 0.6kiB/s
+2020-08-25 13:32:28 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:30 ✨ [Relaychain] Imported #270 (0xe7f2…3743)
+2020-08-25 13:32:30 💤 [Relaychain] Idle (3 peers), best: #270 (0xe7f2…3743), finalized #267 (0x8bfc…8abb), ⬇ 1.2kiB/s ⬆ 0.5kiB/s
+2020-08-25 13:32:33 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:35 💤 [Relaychain] Idle (3 peers), best: #270 (0xe7f2…3743), finalized #267 (0x8bfc…8abb), ⬇ 0.7kiB/s ⬆ 0.3kiB/s
+2020-08-25 13:32:36 ✨ [Relaychain] Imported #271 (0x256f…8849)
+2020-08-25 13:32:38 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:40 💤 [Relaychain] Idle (3 peers), best: #271 (0x256f…8849), finalized #269 (0x2dd2…9964), ⬇ 1.5kiB/s ⬆ 0.5kiB/s
+2020-08-25 13:32:42 ✨ [Relaychain] Imported #272 (0x709a…c025)
+2020-08-25 13:32:43 💤 [Parachain] Idle (0 peers), best: #0 (0x6d45…12f6), finalized #0 (0x6d45…12f6), ⬇ 0 ⬆ 0
+2020-08-25 13:32:45 💤 [Relaychain] Idle (3 peers), best: #272 (0x709a…c025), finalized #269 (0x2dd2…9964), ⬇ 0.6kiB/s ⬆ 0.3kiB/s
+```
```

Comment:
I think we can shorten the output here by 15 - 20 lines and still get the point across..